src/lib/UtapiClient.js

```diff
@@ -207,10 +208,20 @@ export default class UtapiClient {
      * @return {undefined}
      */
     _logMetric(params, method, timestamp, log) {
-        const { bucket, accountId } = params;
-        const logObject = bucket ? { bucket } : { accountId };
-        logObject.method = `UtapiClient.${method}`;
-        logObject.timestamp = timestamp;
+        const { bucket, accountId, user } = params;
+        const logObject = {
+            method: `UtapiClient.${method}`,
+            timestamp,
+        };
+        if (bucket) {
```

Comment:
These tests are not actually useful, if the value is undefined it will just not show up in the logs.